tasks/fh-build.js

```diff
@@ -366,6 +366,25 @@ module.exports = function(grunt) {
                       'shell:fh-run-array:accept']);
     } else if (this.target === 'unit') {
       grunt.task.run(['shell:fh-run-array:unit']);
+    } else if (this.target === 'testfile'){
+      var testFile = arguments[0];
+      if (!testFile){
+        grunt.log.errorlns("Please specify test file to run: grunt fh:testfile:filename.js");
+        return;
+      }
+      // Task name (grunt file config variable)
+      var testType = "unit_single";
+      // Task filename parameter (used as template in runtime script)
+      grunt.config.set("unit_test_filename", testFile);
+      for (var index = 1; index < arguments.length; index++){
+        grunt.config.set("unit_test_param" + index, arguments[index]);
+      }
+      var cmdArray = grunt.config.get(testType);
+      if (!cmdArray){
+        grunt.log.errorlns("No local config for single tests. Please define " + testType + " property in config.");
+        return;
+      }
```

Comment:
grunt.config.get(testType); is very sneaky because apart from getting data it's also forcing new templates to be resolved. All variables we set above are used in that template so we need that in such order. 

is also resolving applied templates again.